### DISCUSSION TITLE
Feature Request: Rounded corner for Wifi and Battery system tray on the windows 10 flyout?

### DISCUSSION THREAD

**Discussion by Bobu5** [valinet/ExplorerPatcher#1025] [Ideas]
Just updated my Explorer Patcher to `22000.493.42.31` from `22000.493.42.28` , just noticed that the wifi system tray doesnt have the rounded edges like it used to be, heres a screenshot of it
![image](https://user-images.githubusercontent.com/87025837/156976688-7be2d40f-f15d-41d6-8fe2-a3450b759433.png)
[the new Wifi system tray]

 is it possible to add an option to toggle it for the wifi and battery system tray? (just like what u guys did to the Windows 10 Start menu corner preference)  Thanks for the time to read this discussion
 
 -Bobu5

**Comment by Bobu5** [comment]
Forgot to mention , im currently using windows 11 21H2
![image](https://user-images.githubusercontent.com/87025837/156978322-9634940e-b740-4252-b015-4565cc7f58da.png)
![image](https://user-images.githubusercontent.com/87025837/156978347-def34bbf-5956-432e-a488-77ffe2a2ae86.png)

**Comment by Poopooracoocoo** [comment]
Not related to your feature request:
>wifi system tray doesnt have the rounded edges _like it used to be_

Hm. Maybe you had the network icon set to open the Windows 11 flyout and it got overridden?
![image](https://user-images.githubusercontent.com/51363864/156978579-f933fe21-8df4-42c3-ae02-c2f4bc9df456.png)

I don't think there was ever a commit that added rounded corners to the Windows 10 network flyout

**Reply by Bobu5** [reply]
> I don't think there was ever a commit that added rounded corners to the Windows 10 network flyout

huh, i remember having my Wifi's tray having these additional option 
![image](https://user-images.githubusercontent.com/87025837/156979962-0c64be92-960a-4ee7-affa-ff0b732422b1.png)
but with the windows 11 wifi tray ui
![image](https://user-images.githubusercontent.com/87025837/156980016-af9245ef-ac59-4332-bd95-2c80f1fb8292.png)

**Reply by Bobu5** [reply]
quite strange ig
